pandas/tests/series/test_operators.py

```diff
@@ -1613,7 +1602,7 @@ def timedelta64(*args):
 
             try:
                 assert_series_equal(lhs, rhs)
-            except:
```

Comment:
what the heck is this? don't try/except and then raise an assertion :<

Reply:
I can only speculate that the original author wanted to customize the exception message.

Reply:
pls remove it’s not necessary
the asserts all do this now